### PR TITLE
feat: message lightbox added to lender profile

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- message lightbox added to lender profile

Not signed:
Screenshot 2024-08-01 at 2 48 23 p m

Signed and no loans:
Screenshot 2024-08-01 at 2 51 37 p m

Signed with loans:
Screenshot 2024-08-01 at 2 47 25 p m

### CODE REVIEW

File: src/components/LenderProfile/LenderSummary.vue [hunk 1]
```diff
@@ -53,20 +60,97 @@
...
+			:title="lightboxTitle"
+		>
+			<!-- eslint-disable max-len vue/singleline-html-element-content-newline -->
+			<template v-if="!isLoggedIn">
+				<p>
+					You must be signed in and have made loans on Kiva first to send a Lender Message. This helps us reduce the amount of SPAM and unwanted messages in the system.
+				</p>
+			</template>
+
+			<template v-if="isLoggedIn && loansNumber == 0">
```
Comment (dyersituations) [inline src/components/LenderProfile/LenderSummary.vue:75]: I think we'll need this to be `v-else-if` for the Vue 3 migration. Same with below on line 86.

File: src/components/LenderProfile/LenderSummary.vue [hunk 2]
```diff
@@ -53,20 +60,97 @@
...
+				<p>In order to send a message, you will need to make a loan using your own funds first. This helps us prevent spam and unwanted messages.</p>
+				<p class="tw-mt-2">
+					Please
+					<router-link to="/lend">
```
Comment (dyersituations) [inline src/components/LenderProfile/LenderSummary.vue:79]: Analytics?

File: src/components/LenderProfile/LenderSummary.vue [hunk 3]
```diff
@@ -53,20 +60,97 @@
...
+							v-model="lenderMessage"
+						>
+					</textarea>
+						<kv-material-icon
+							class="tw-w-2.5 tw-h-2.5 tw-absolute tw-bottom-3 tw-right-1"
+							:icon="mdiPencilOutline"
+						/>
+					</div>
+					<div class="tw-flex tw-flex-col lg:tw-flex-row tw-gap-2">
+						<kv-button
```
Comment (dyersituations) [inline src/components/LenderProfile/LenderSummary.vue:110]: Analytics on these two buttons?

File: src/components/LenderProfile/LenderSummary.vue [hunk 4]
```diff
@@ -53,20 +60,97 @@
...
+							variant="secondary"
+							@click="closeLightbox"
+						>
+							Close
+						</kv-button>
+					</div>
+				</div>
+				<p class="tw-mt-4">
+					Lender messages shouldn't be used for self-promotion, advertising or solicitation. More information can be found in
+					<router-link to="/kiva-community-guidelines">Kiva's community guidelines</router-link>.
```
Comment (dyersituations) [inline src/components/LenderProfile/LenderSummary.vue:129]: Analytics?

Comment (roger-in-kiva) [reply src/components/LenderProfile/LenderSummary.vue:129]: We have other ticket for all analytics events on the page: https://kiva.atlassian.net/browse/MP-570

**Review by dyersituations — APPROVED** [review]

**Pull request merged**